modules/planning/tasks/deciders/path_reference_decider/path_reference_decider.h

```diff
@@ -109,6 +109,10 @@ class PathReferenceDecider : public Task {
   bool IsADCBoxAlongPathReferenceWithinPathBounds(
       const std::vector<common::TrajectoryPoint> &path_reference,
       const PathBoundary *regular_path_bound);
+
+ private:
+  static int valid_path_reference_counter_;  // count valid path reference
+  static int total_path_counter_;            // count total path
```

Comment:
what are these two member variables used for? just for debugging/statistics purpose?

Reply:
Yes. They are. For Now.